apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -132,7 +132,7 @@ export class BulkScriptAction extends BaseScriptAction {
     });
 
     return taskRunner
-      .execute()
+      .executeAsync()
       .then(() => {
```

Comment:
Can we convert this to `async`/`await`?

Reply:
Good catch -- that function was apparently already halfway converted